src/emailpro/account/alias/update/emailpro-account-alias-update.html

```diff
@@ -16,7 +16,7 @@
             </div>
 
             <div class="row pull-left mtop70" data-ng-show="noDomainMessage">
```

Comment:
Seems that update function isn't used anymore. Folder and file could be removed.

Reply:
Fixed ! Deleted file !